src/core/dfn-abbr.js

```diff
@@ -0,0 +1,49 @@
+// @ts-check
+// Module core/dfn
+// - Finds all <dfn> elements with data-abbr attribute and parses them and registers definitions.
+// - Also populates abbrMap used by core/inlines for such definitions.
+
+import { registerDefinition } from "./dfn-map";
+
+export const name = "core/dfn-abbr";
+export const abbrMap = new Map();
+
+/**
+ * Generates simple Abbreviations from text passed.
+ *
+ * @param {String} text A string like "Permanent Account Number".
+ * @returns {String} abbr Abbreviation like "PAN"
+ */
+function getAbbreviationFromText(text) {
```

Comment:
This can be inlined too.

Reply:
@sidvishnoi suggests otherwise. 
https://github.com/w3c/respec/pull/2189#discussion_r268429359

Reply:
I don't mind... but I think we can communicate that just as effectively with a code comment there.

Reply:
Inlined. :smiley: